src/main/java/no/sikt/sws/ListIndecesHandler.java

```diff
@@ -0,0 +1,52 @@
+package no.sikt.sws;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.sikt.sws.exception.SearchException;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ListIndecesHandler extends ApiGatewayHandler<String, String> {
+
+    public static final String RESOURCE_IDENTIFIER = "resource";
+    public OpenSearchClient openSearchClient = new OpenSearchClient();
+
+
+    private static final Logger logger = LoggerFactory.getLogger(ListIndecesHandler.class);
+
+    public ListIndecesHandler() {
+        super(String.class);
+    }
+
+    @Override
+    protected String processInput(
+            String body,
+            RequestInfo request,
+            Context context
+    ) throws ApiGatewayException {
+
+        var httpMethod = RequestUtil.getRequestHttpMethod(request);
```

Comment:
Not needed. Just hardcode it as a GET when you communicate with the openSearchClient
The template and openapi files wont allow anything else